### PR TITLE
Adds the meown shoes to the contraband section of the AutoDrobe.

### PR DESCRIPTION
**Pull request by dopamiin0**
## About The Pull Request
Adds the "meown shoes" to the contraband section of AutoDrobes. "Meown shoes" are cute little cat slippers that meow every single time you step.

Sound source: https://freesound.org/people/swatkamus/sounds/260179/
## Why It's Good For The Game

https://www.youtube.com/watch?v=Bze2Cl84wRg

## Changelog

:cl: dopamiin and tattle
add: Distant meowing can be heard from the restricted section of station AutoDrobes.
soundadd: A small cat meow designed to be played about three times in one second, deafening innocent crew.
imageadd: Cute cat slipper sprites.
/:cl:

### CODE REVIEW

File: code/modules/clothing/shoes/clown.dm [hunk 1]
```diff
@@ -5,14 +5,15 @@
 	inhand_icon_state = "clown_shoes"
 	slowdown = SHOES_SLOWDOWN+1
 	var/enabled_waddle = TRUE
+	var/list/squeak_sound = list('sound/effects/clownstep1.ogg'=1,'sound/effects/clownstep2.ogg'=1) //allows clownshoe subtypes to have different sounds
```
Comment (san7890) [inline code/modules/clothing/shoes/clown.dm:8]:
```suggestion
    ///List of sounds we can choose from on movement of the clown.
	var/list/squeak_sound = list('sound/effects/clownstep1.ogg'=1,'sound/effects/clownstep2.ogg'=1) 
```

Would you mind [dmdoccing](https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md) this? You just have to write the comment, then the variable as such. This allows the language server and codedocs.tgstation13.org to be able to parse it. I wrote an example one for you, but it's likely that this suggestion will probably break your code somehow. I would do a similar edit on your local client (moving the comment up a line, being a bit more general in the statement, and prefixing the comment with `///`).

Thank you!

Comment (dopamiin0) [reply code/modules/clothing/shoes/clown.dm]: i think i did it but let me know if that's wrong

Comment (san7890) [reply code/modules/clothing/shoes/clown.dm]: Looks much better to my eyes now, thank you again!

**Comment by bluetshirtguy** [comment]
are they valid?

**Comment by dopamiin0** [comment]
> are they valid?

if you mean this as in "is it a valid item," I don't know why it wouldn't be. i've tested this several times with no errors. if you mean this as in "are you valid to be killed for wearing these," mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew mew

**Comment by SuperSlayer0** [comment]
Cringe

**Review by AnturK — APPROVED** [review]

**Review by tralezab — APPROVED** [review]

**Comment by optimumtact** [comment]
> Cringe

Ay man this you?

**Comment by SuperSlayer0** [comment]
> Ay man this you?
> 

Yes.

**Comment by dopamiin0** [comment]
LOL

**Comment by SuperSlayer0** [comment]
> Ay man this you?
> 

Heard about such thing as "humour"?

**Pull request merged**